src/sdk/helpers/smil-document-parser.js

```diff
@@ -0,0 +1,248 @@
+//  Copyright (c) 2014 Readium Foundation and/or its licensees. All rights reserved.
```

Comment:
We'll have to normailse the spaces here. [js-beautify](https://www.npmjs.com/package/js-beautify) comes in handy for that.